tests/cores/mega/test_multimodalqna_gateway.py

```diff
@@ -72,21 +74,30 @@ async def lvm_add(request: Union[LVMDoc, LVMSearchedMultimodalDoc]) -> TextDoc:
     return res
 
 
+@register_microservice(name="asr", host="0.0.0.0", port=3001, endpoint="/v1/audio/transcriptions")
```

Comment:
Use another port that's not the real ASR port. For example they are already using 8083, 8084, 8085 for the other microservices, so we could use 8086 for ASR.

```suggestion
@register_microservice(name="asr", host="0.0.0.0", port=8086, endpoint="/v1/audio/transcriptions")
```